client/flutter/lib/components/page_scaffold/page_scaffold.dart

```diff
@@ -14,28 +20,44 @@ class PageScaffold extends StatelessWidget {
     @required this.body,
     this.title,
     this.heroTag,
-    this.padding = EdgeInsets.zero,
+    this.header,
     this.showHeader = true,
+    this.padding = EdgeInsets.zero,
+    this.color = CupertinoColors.white,
+    this.headingBorderColor = const Color(0xffC9CDD6),
+    this.beforeHeader = const <Widget>[],
+    this.disableBackButton = false,
+    this.headerFontSize = 34,
   });
 
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: this.padding,
-      child: Stack(
-        children: <Widget>[
-          CustomScrollView(slivers: [
-            if (this.showHeader)
-              PageHeader(
-                title: this.title,
-                heroTag: this.heroTag ?? this.title,
-              ),
-            ...this.body,
-            SliverToBoxAdapter(
-              child: SizedBox(height: 70),
-            ),
-          ]),
-        ],
+    return Container(
+      color: this.color,
+      child: Padding(
+        padding: this.padding,
+        child: Stack(
+          children: <Widget>[
+            CustomScrollView(
+                physics: BouncingScrollPhysics(parent: ClampingScrollPhysics()),
+                slivers: [
+                  ...this.beforeHeader,
```

Comment:
I get what you're doing here with the `beforeHeader`, but... on the pages where we have the promo content, is the page title really still a header? I would have thought about those pages as not having a header at all, and the title & promo content just being part of the page `body` (but, as always, open to hearing why that approach is wrong 😄)

Reply:
Hmm... it says you hid this comment so not sure if you don't want me to respond?

I think it is a header, semantically (i.e. from an accessibility perspective).

Reply:
GitHub hid this automatically, it wasn't something I chose - probably GitHub telling me that I was leaving too many comments 😉 

I guess it is hard to tell with the designs not yet finalized what these views might look like and whether those elements will continue to actually be headers, but as they stand right now, they are, and the accessibility argument makes sense to me 👍 

Now I'll mark this as resolved to hide it 😄